docs/integrations/ml-dist.rst

```diff
@@ -0,0 +1,512 @@
+vary as well.
+
+Saving A Model To S3
+--------------------
+
+The next step is to save the model to a AWS S3 Bucket so we can access the model from
+potentially another application. Since the data we want to predict will not be normalised
+like the model, we also need to save the MinMaxScaler we created before alongside the
+TensorFlow model.
+
+.. code-block:: python
+
+    import os
+    import boto3
+    from joblib import dump
+
+
+    bucket = "some-s3-bucket-name"
+
+    model_name = "some-model-name" + ".h5"
+    mlp_model.save(model_name)
+
+    scaler_name = "some-scaler-name" + "_scaler.pkl"
+    dump(min_max_scaler, scaler_name)
+
+    client = boto3.client("s3")
+    client.upload_file(model_name, bucket, model_name)
+    client.upload_file(scaler_name, bucket, scaler_name)
+
+    os.remove(model_name)
+    os.remove(scaler_name)
+
+Now we should see our model and scaler in the according S3 bucket and we can continue
+with loading it from possibly a different application.
+
+Loading A Model From S3
+-----------------------
+
+Now that the model is saved in an AWS S3 Bucket, we can simply load it from there
+and use it to predict pump sensor data without having to first train a model.
+
+.. code-block:: python
+
+    bucket = "some-s3-bucket-name"
+
+    model_name = "some-model-name"
+    scaler_name = "some-scaler-name"
+
+    client = boto3.client("s3")
+    client.download_file(bucket, model_name, model_name)
+    client.download_file(bucket, scaler_name, scaler_name)
+
+
+Predicting With Pre-Trained Model
+---------------------------------
+
+The models are now available, this means we can load them from file and start preparing
+the data to predict it. For simplicity we will take the same data that we used to
+train and test the model.
+
+.. code-block:: python
+
+    import tensorflow as tf
+    from joblib import load
+
+
+    model = tf.keras.models.load_model(model_name)
+    scaler = load(scaler_name)
+
+    # We can use the df we queried in our very first step
+    new_data = df
+
+    data_cols = df.iloc[:, 2:54]
+    norm_data_cols = scaler.transform(data_cols)
+
+    pred_pairs = []
+    # the timestep is still the same as in the earlier part
+    pred_saved_step = 0
+    for i in norm_data_cols.index:
+        row = pred_saved_step
+        if row + timestep < len(df_train.index):
+            data_window = norm_data_cols[row:row + timestep]
+            data_vector = data_window.flatten()
+            pred_pairs.append(data_vector)
+            pred_saved_step = row + timestep
+        else:
+            break
+
+    predictions = model.predict(pred_pairs.reshape(1, -1))
+    predictions.flatten()
+
+    [1] [0.8614458441734314, 0.8530051112174988, 0.8502672910690308, 0.8750132918357849, 0.8636448979377747, ...]
+
+Here we can se the first few predicted values, everything > 0.5 would mean that the machine is in a
+normal state, while < 0.5 would mean it is broken.
```

Comment:
```suggestion
We now have our predictions. Predictions > 0.5 mean that the machine is in a
normal state, while < 0.5 mean it is broken.
```
Do you know what prediction == 0.5 (unlikely as that is) would be? Should we say that everything ≥ 0.5 is normal, or is that inaccurate?